packages/core/src/QueryProviders/QueryProviderSundaeSwap.ts

```diff
@@ -343,23 +487,33 @@ export class QueryProviderSundaeSwap implements QueryProvider {
           bReserve: BigInt(pool.current.quantityB.quantity ?? 0),
           lpTotal: BigInt(pool.current.quantityLP.quantity ?? 0),
         },
+        linearAmplificationFactor: BigInt(pool.linearAmplificationFactor),
+        protocolFee: new Fraction(...pool.protocolAskFee).toNumber(),
         version: pool.version,
       };
     });
   }
 
   async findPoolData(identArgs: IPoolByIdentQuery): Promise<IPoolData>;
   async findPoolData(assetArgs: IPoolByAssetQuery): Promise<IPoolData[]>;
+  async findPoolData(assetPairArgs: IPoolByPairQuery): Promise<IPoolData[]>;
   async findPoolData(searchArgs: IPoolBySearchTermQuery): Promise<IPoolData[]>;
   async findPoolData(
-    args: IPoolByIdentQuery | IPoolByAssetQuery | IPoolBySearchTermQuery,
+    args:
+      | IPoolByIdentQuery
+      | IPoolByAssetQuery
+      | IPoolBySearchTermQuery
+      | IPoolByPairQuery,
```

Comment:
We should really update these types to be dependent on a union, so we know what the args are.